docker-compose.yml

```diff
@@ -1,116 +1,124 @@
-# By default, this docker-compose setup uses Kafka 2.4.1. This version can
-# be overwritten by setting the KAFKA_IMAGE_TAG environment variable; some choices here include:
+# By default, this docker-compose setup uses Kafka 2.7.0
 #
-# 1. Kafka 2.6: 2.13-2.6.0
-# 2. Kafka 2.5: 2.13-2.5.0
-# 3. Kafka 0.10: 2.11-0.10.2.2
-#
-# See https://hub.docker.com/r/wurstmeister/kafka/tags for the complete list.
-version: '2.1'
+# See https://hub.docker.com/r/bitnami/kafka/tags for the complete list.
+version: '3'
 services:
   zookeeper:
-    image: wurstmeister/zookeeper
+    container_name: zookeeper
+    hostname: zookeeper
+    image: bitnami/zookeeper:latest
     ports:
       - "2181:2181"
+    environment:
+      ALLOW_ANONYMOUS_LOGIN: yes
 
   # Zone 1 brokers
   kafka1:
-    image: wurstmeister/kafka:${KAFKA_IMAGE_TAG:-2.12-2.4.1}
+    container_name: kafka1
+    hostname: 169.254.123.123
+    image: bitnami/kafka:2.7.0
```

Comment:
Are we dropping support to other versions here by not using `${KAFKA_IMAGE_TAG}` ?

Reply:
This has been included in recent commit